### PR TITLE
GraphQL: Optimize on Relation (and Prettier stuff) 

### PR DESCRIPTION
**Pull request by Moumouls**
Strange, it seems that some files have not been processed by prettier. 

Main fix is here

```js
const needToGetAllKeys = (fields, keys, parseClasses) =>
  keys
    ? keys.split(',').some(keyName => {
        const key = keyName.split('.');
        if (fields[key[0]]) {
// We miss the Relation optimization
          if (fields[key[0]].type === 'Relation') return false;
          if (fields[key[0]].type === 'Pointer') {
            const subClass = parseClasses.find(
              ({ className: parseClassName }) => fields[key[0]].targetClass === parseClassName
            );
            if (subClass && subClass.fields[key[1]]) {
              // Current sub key is not custom
              return false;
            }
          } else if (
            !key[1] ||
            fields[key[0]].type === 'Array' ||
            fields[key[0]].type === 'Object'
          ) {
            // current key is not custom
            return false;
          }
        }
        // Key not found into Parse Schema so it's custom
        return true;
      })
    : true;
```

### CODE REVIEW

**Comment by davimacedo** [comment]
@Moumouls I can't find the changes in the PR. You only pushed the prettier :)

**Comment by Moumouls** [comment]
@davimacedo  change is ` if (fields[key[0]].type === 'Relation') return false;`

File: src/GraphQL/helpers/objectsQueries.js [hunk 1]
```diff
@@ -10,10 +10,10 @@ const needToGetAllKeys = (fields, keys, parseClasses) =>
     ? keys.split(',').some(keyName => {
         const key = keyName.split('.');
         if (fields[key[0]]) {
+          if (fields[key[0]].type === 'Relation') return false;
```
Comment (Moumouls) [inline src/GraphQL/helpers/objectsQueries.js:13]: Herer @davimacedo

**Review by davimacedo — APPROVED** [review]
Sorry. My bad. LGTM!

**Pull request merged**